src/rpc/protocol/query.rs

```diff
@@ -784,3 +823,45 @@ impl ProtoDecoder for ObTableStreamRequest {
         unimplemented!()
     }
 }
+
+#[derive(Default, Debug, Clone, PartialEq)]
+pub enum ObTableAggregationType {
+    #[default]
+    INVAILD = 0,
+    MAX = 1,
+    MIN = 2,
+    COUNT = 3,
+    SUM = 4,
+    AVG = 5,
+}
+
+#[derive(Default, Debug, Clone, PartialEq)]
+pub struct ObTableAggregationSingle {
```

Comment:
Do you think **ObTableAggregationOp** is a better name?
We could comment that this struct is a single aggregation.

Reply:
fixed